src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private static final String OUTPUT_SQUARE = "Figure: square, "
+            + "area: %.2f sq. units, "
+            + "side: %.2f units, "
+            + "color: %s";
+    private double side;
+
+    public Square() {
+        super();
```

Comment:
this is redundant, Java does it automatically 
_If a constructor does not explicitly invoke a superclass constructor, the Java compiler automatically inserts a call to the no-argument constructor of the superclass_